CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Move all commonly used imports into all_components.scss (PR #683)
```

Comment:
This has already been released - I think you need a rebase.

Reply:
@andysellick - rebased